packaging/images/openshift-ci/Dockerfile.test-runtime

```diff
@@ -1,3 +1,4 @@
+# hadolint global ignore=DL3002,DL4006,SC3037,SC2086
```

Comment:
```
DL3002 warning: Last USER should not be root
SC3037 warning: In POSIX sh, echo flags are undefined.
SC2086 info: Double quote to prevent globbing and word splitting.
DL4006 warning: Set the SHELL option -o pipefail before RUN with a pipe in it.
```

Reply:
I think we may want to move the first warning to the global flags and fix the other 3 in the file.

Reply:
Let's try to avoid having local suppressions.

Reply:
- DL3002 : Moved to common config
- SC3037 : replaced `echo -e` with `printf` with some formatting changes
- SC2086 : Added Double quote
- DL4006: Added `SHELL ["/bin/bash", "-o", "pipefail", "-c"]`   Ref [here](https://github.com/hadolint/hadolint/wiki/DL4006)